### PR TITLE
ENH: show seconds from timestamp in the filename

### PR DESCRIPTION
**Pull request by chiahaoliu**
fix issue 262 at ``xpdAcq``

### CODE REVIEW

File: xpdan/utils.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def _clean_info(input_str):
 def _timestampstr(timestamp):
     """ convert timestamp to strftime formate """
     timestring = datetime.datetime.fromtimestamp(float(timestamp)).strftime(
-        '%Y%m%d-%H%M')
+            '%Y%m%d-%H%M:%S')
```
Comment (CJ-Wright) [inline xpdan/utils.py]: Is colon ok for a filename? (Will this work on windows?)

Comment (chiahaoliu) [inline xpdan/utils.py]: good point. I checked,  it isn't. I think we might want to use ``-`` then? 

``_`` has been used to separate different parts of information

**Comment by sbillinge** [comment]
ok '-' is fine, or just nothing.  It is unambiguous if there are always two
digits for each, so 10:15:32 would just be 101532.  With your suggestion it
would be 1015-32, which looks a bit odd because the seconds suddenly have
different status.

With no delimiter it would be, in full 20161013-101532, which is a bit hard
to digest but gets the job done, and can be used to auto-sort on.

S

**Comment by chiahaoliu** [comment]
okay, no special character in between seems to be a better solution.

**Pull request merged**